app/jiraapi.py

```diff
@@ -139,35 +149,44 @@ def create_domain_approval_task(self, assessment, application=None):
         if supplier.is_recruiter == 'true' and case_study_links == '':
             summary += '[RECRUITER]'
         elif supplier.is_recruiter == 'false' or case_study_links != '':
-            price_approved = True
-            maxPrice = ''
-            try:
-                pricing = supplier.data.get('pricing', None)
-                if pricing:
-                    maxPrice = pricing[domain_name]['maxPrice']
-            except KeyError:
-                pass
+            price_status = None
+            domain_price_minimum = domain.price_minimum
+            domain_price_maximum = domain.price_maximum
+            max_price = None
+
             if application:
                 pricing = application.data.get('pricing', None)
                 if pricing:
                     domain_pricing = pricing[domain_name]
                     if domain_pricing:
-                        price_approved = False
-                        maxPrice = domain_pricing = domain_pricing['maxPrice']
+                        max_price = int(domain_pricing['maxPrice'])
+            elif 'pricing' in supplier.data:
+                supplier_pricing = supplier.data.get('pricing', None)
+                if domain_name in supplier_pricing:
+                    max_price = int(supplier_pricing[domain_name]['maxPrice'])
```

Comment:
Does this need to cater for decimal pricing? similar to https://github.com/AusDTO/dto-digitalmarketplace-api/commit/6ea0cfca4fa348d6d74020dfe5d00d532b21a821

Reply:
good one: fixed